### DISCUSSION TITLE
CM5 Drop in Replacement

### DISCUSSION THREAD

**Discussion by MartyF81** [WLAN-Pi/feedback#80] [Hardware - WLAN Pi M4] [answered]
I have the M4... I am wondering if the CM5 with WiFi module... would be a drop in replacement? Would like to add a little bit more horsepower by getting an 8GB RAM CM5.

**Comment by jiribrejcha** [comment, marked as answer]
No, we don’t recommend that.

Our current software image doesn’t support the CM5.

From hardware perspective, it is untested and you will be walking on your own. While the CM connectors are backwards compatible, the CM5 runs significantly warmer. It requires a fan or much larger heat sink. And that might not even fit inside the M4 case.
